client/components/VotingButtons.jsx

```diff
@@ -8,19 +8,25 @@ function Voting({ cat, dog }) {
   const dispatch = useDispatch()
 
   // win 2 points
-  // draw 1 point
   function handleSubmit(winnerId) {
-    if (winnerId !== 'skip') {
-      postVotes(winnerId)
-        .then(() => {
-          dispatch(fetchTwoPets())
-        })
-        .catch((err) => {
-          console.log(err)
-        })
-    } else {
-      dispatch(fetchTwoPets())
-    }
+    postVotes(winnerId)
+      .then(() => {
+        dispatch(fetchTwoPets())
+      })
+      .catch((err) => {
+        console.log(err)
```

Comment:
this code isn't covered - please add a test for when the postVotes promise rejects